src/Stores/Subjects/actions.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+import type { Dispatch, Subject, SubjectId, PromiseId } from '../../types';
+import {
+  getSubjects as apiGetSubjects,
+  linkSubjectPromise as apiLinkSubjectPromise,
+} from '../../utils/api';
+
+const getSubjects = () => ({ type: 'SUBJECTS_LOAD' });
+
+const getSubjectsSuccess = (subjects: Array<Subject>) => ({
+  subjects,
+  type: 'SUBJECTS_LOAD_SUCCESS',
```

Comment:
Move types into their own file so that the actions and reducers both use the same variable? Instead of using hardcoded strings

Reply:
Good suggestion.
I'll give it a stab